File: C/dentro_fora.c
#include <stdio.h>

int main()
{
    int n, num, in, out;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);

    in = 0;
    out = 0;
    for (int i = 1; i <= n; i++)
    {
        printf("Digite um numero: ");
        scanf("%d", &num);
        if (num >= 10 && num <= 20)
        {
            in++;
        }
        else
        {
            out++;
        }
    }
    printf("%d DENTRO\n", in);
    printf("%d FORA", out);
    return 0;
}

File: Python/dentro_fora.py
dentro: int; fora: int

n = int(input("Quantos numeros voce vai digitar? "))
dentro = 0
fora = 0

for i in range(0, n):
    x = int(input("Digite um numero: "))
    if x >= 10 and x <= 20:
        dentro = dentro + 1
    else:
        fora = fora + 1

print(f"{dentro} DENTRO")
print(f"{fora} FORA")
